moduleroot/CONTRIBUTING.md.erb

```diff
@@ -174,6 +174,26 @@ and so on. rspec tests may have the same kind of dependencies as the
 module they are testing. While the module defines in its [Modulefile](./Modulefile),
```

Comment:
`Modulefile` hasn't been a thing for some time and has been replaced with `metadata.json`.

Reply:
good point, fixed it.